src/NexusMods.App.UI/Windows/MainWindowViewModel.cs

```diff
@@ -118,6 +122,9 @@ public MainWindowViewModel(
                     var workspaceController = self.WorkspaceController;
                     if (workspaceController.ActiveWorkspace.Context is not LoadoutContext loadoutContext) return;
 
+                    var loadout = Loadout.Load(self._connection.Db, loadoutContext.LoadoutId);
+                    if (!loadout.IsValid() || loadout.Installation.GameId != message.Revision.Collection.GameId) return;
+
```

Comment:
Shouldn't we be opening the relevant workspace for the user?

Reply:
If the currently open loadout matches the game, we open there, if not, either open the last applied loadout for that game or the first one if that data is hard to get.

Reply:
Updated.